gef.py

```diff
@@ -8861,9 +8877,11 @@ def do_invoke(self, argv):
             insnum_step = 1
 
         start_address = align_address(addr)
+        ref_address = align_address(ref_addr)
+        base_offset = start_address-ref_address
```

Comment:
here the base_offset is calculated as the difference between the start_address and the reference address

Reply:
Nit: spaces around operators